doc/sphinx/source/installation.rst

```diff
@@ -4,31 +4,4 @@
 Installation
```

Comment:
Might as well remove this directly?

Reply:
I think yes but it will probably have to happen within a future release, unaware of this used anywhere.